var/spack/repos/builtin/packages/icedtea/package.py

```diff
@@ -139,6 +141,19 @@ class Icedtea(AutotoolsPackage):
              url='http://icedtea.wildebeest.org/download/drops/icedtea8/3.4.0/shenandoah.tar.xz',
              when='@3.4.0')
 
+    # FIXME:
+    # 1. `extends('java')` doesn't work, you need to use `extends('icedtea')`
+    # 2. Packages cannot extend multiple packages, see #987
+    # 3. Override activate and deactivate to symlink all .jar files to
```

Comment:
@adamjstewart could you rewrite (3) according to https://github.com/spack/spack/pull/8613#issuecomment-412738031 (or use the wording suggested there)? The way it's worded now, it looks like something a package author could manage, but I think it will actually take some work on the core (e.g. updating `PackageViewMixin` to add a symlink to the jar files in `lib/ext`).

Other than that this looks good.